packages/test/end-to-end-tests/src/test/oldVersion.ts

```diff
@@ -56,166 +49,6 @@ import {
 import * as newVer from "./newVersion";
 /* eslint-enable import/no-extraneous-dependencies */
 
-const defaultCodeDetails: IFluidCodeDetails = {
```

Comment:
Can you put a comment in `oldVersion2.ts` that this should be removed there as well as part of the version bump?